assets/images/lets-chat.svg

```diff
@@ -0,0 +1 @@
+<svg width="125" height="59" viewBox="0 0 125 59" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M94.5 25.7v7.9" stroke="#0B1B34" stroke-width="2" stroke-miterlimit="10"/><path d="M118.7 51c.1-11.1-10.8-20.2-24.2-20.2-13.4 0-24.3 9-24.2 20.2h48.4Z" fill="#0185FF" stroke="#fff" stroke-width="2" stroke-miterlimit="10"/><path d="M112.1 45.1c-1.6-3.4-4.7-6.2-8.5-8l8.5 8Z" fill="#0B1B34"/><path d="M112.1 45.1c-1.6-3.4-4.7-6.2-8.5-8" stroke="#fff" stroke-width="2" stroke-miterlimit="10" stroke-linecap="round" stroke-linejoin="round"/><path d="M98.8 26.7c0-2-2-3.6-4.4-3.6-2.4 0-4.4 1.6-4.4 3.6h8.8Z" fill="#0B1B34"/><path d="M122.4 58H66.5l3.7-6.9h48.5l3.7 6.9Z" fill="#0185FF" stroke="#fff" stroke-width="2" stroke-miterlimit="10"/><path d="M94.1 10.2v7.1M83 13.2l4 5.1M105.2 13.2l-4 5.1" stroke="#0B1B34" stroke-width="2" stroke-miterlimit="10"/><path d="M1.5 23.6c0 12.5 10.1 22.6 22.6 22.6 5.4 0 10.3-1.9 14.2-5h18.4L45.5 30.8c.8-2.3 1.2-4.7 1.2-7.2C46.7 11.1 36.6 1 24.1 1 11.6 1 1.5 11.1 1.5 23.6Z" fill="#0185FF" stroke="#fff" stroke-width="2" stroke-miterlimit="10" stroke-linejoin="round"/><path d="M21.9 11.6h4.6v5.7l-1.2 10.4h-2.2l-1.2-10.4v-5.7ZM24.2 34.5a2.3 2.3 0 1 0 0-4.6 2.3 2.3 0 0 0 0 4.6Z" fill="#fff"/></svg>
```

Comment:
Just confirming that in my Figma mockup, I have this image at 160x100:
![image](https://user-images.githubusercontent.com/2319350/135217956-41f16b89-0869-47c4-a0dc-99f029a54a6f.png)

Just curious where this image is coming from as it seems like the size is different?

Reply:
Here is my Figma version: [lets_chat.svg.zip](https://github.com/Expensify/App/files/7249513/lets_chat.svg.zip)

Reply:
I'll update that! I got it from Figma, but I selected the layer that doesn't have the spacing to export it.